features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Applications/Listing/AppsTableContent.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+"use strict";
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import {TableBody, TableCell, TableRow} from 'material-ui/Table';
+import DeleteIcon from 'material-ui-icons/Delete';
+import IconButton from 'material-ui/IconButton';
+import Tooltip from 'material-ui/Tooltip';
+import {CircularProgress} from 'material-ui/Progress';
+
+const AppsTableBody = (props) => {
+    const {apps, handleAppDelete, classes} = props;
+    let appsTableData = [];
+    apps.forEach(app => {
```

Comment:
Can use `.map` method here.

~~~javascript
const appsTableData = apps.map(app => (
  <TableRow hover tabIndex="-1" key={app.applicationId}>
     <TableCell>..
~~~

Reply:
Thanks for the comment @roonyh , Here apps is an instance of [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map)  which does not have an [Array.map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) alike method to map entries into new Map object. So that I have used the `.forEach` method in Map.

Reply:
Ah. got it. thanks :)